### DISCUSSION TITLE
Black screen Log window

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#614] [General]
Hi,

Very strange. I got a black screen in the admin area when I try to go to the log section. 

I tried:
- restart HFS
- Tried another browser, but in both I got the same
- Tried other section, but it happens only when I trie to go to the log section

Next thing I will try is to boot my computer.

UPDATE:
Booting the computer did not help. Still a black browser window if I go to the log section in the admin area. Yesterday when shut down th computer all was OK.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/c63ffea3-d689-48bb-9cd9-e4dd9f4c0067)

**Comment by rejetto** [comment]
it's surely a bug.
Some line in the log is not correctly handled, and disrupt the interface.
If I had this line I could fix it, but without it's almost impossible.
You can anyway try to collect some info in the browser's console. Sometimes it can be enough.

**Comment by PH5HP** [comment]
Hi Rejetto,

Ah... Why did I not think of that ... F12 in Google Chrome has been my friend several times. PSE see my screenshot.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/54495d86-7bdd-4d3c-b915-569c7c960c97)

**Comment by rejetto** [comment]
that's probably enough, thanks.
let's continue in issue #615
